util/cron/nightly

```diff
@@ -297,6 +297,25 @@ if (! (-r $logdir and -w $logdir and -d $logdir)) {
     print "Error: CHPL_NIGHTLY_LOGDIR ($logdir) not accessible\n";
 }
 
+$memleaksdir = $ENV{'CHPL_NIGHTLY_MEMLEAKS_DIR'};
+
+if ($memleaks ne "") {
+    # Default to known file share.
+    if ($memleaksdir eq "") {
+        $memleaksdir = "/data/sea/chapel/NightlyMemLeaks";
```

Comment:
What do you think about having memleaks default to CHPL_NIGHTLY_LOGDIR in the event that it's not set in order to avoid hardcoding a path into this script (while also preserving the previous behavior).